lib/analytics-page/addon/components/analytics-charts/x-chart-wrapper/styles.scss

```diff
@@ -60,6 +60,10 @@
     margin-bottom: 0;
     font-size: 16px;
     color: inherit;
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    max-width: 90%;
```

Comment:
It looks like `.panel-title` is the class used for `Popular pages` and other chart titles, not the axis labels. I'm not super familiar with the library that we are using to make these graphs, but perhaps there's something in `c3ChartConfig` that we can use to truncate the labels?

Reply:
I'm not [seeeeing](https://c3js.org/reference.html) anything. Maybe it does this automatically if the label is long enough?

Reply:
Looks like it does it automagically!